Makefile

```
@@ -480,6 +480,9 @@ docker/build:
 
 # Adds the current Makefile working directory as a bind mount
 ## Runs the tardigrade-ci docker image
+docker/run/%:
+	@echo "[$@]: Running docker image target=$*"
+	@$(MAKE) docker/run target=$*
```

Let's use `$(SELF)` to ensure we get the right options...
```suggestion
	@$(SELF) docker/run target=$*
```